types/google-apps-script/google-apps-script.addon-event-objects.d.ts

```diff
@@ -0,0 +1,233 @@
+// Type definitions for Google Apps Script Event Objects 2021-04-15
+// Project: https://developers.google.com/apps-script/
+// Definitions by: Oleg Valter <https://github.com/Oaphi>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference path="google-apps-script.script.d.ts" />
+
+declare namespace GoogleAppsScript {
+    /**
+     * @summary Apps Script Addon Event Objects
+     * @see https://developers.google.com/workspace/add-ons/concepts/event-objects
+     */
+    namespace AddonEventObjects {
```

Comment:
IMO: `GoogleAppsScript.AddonEventObjects.EventObject` is lengthy.
How about below candiates?

- `GoogleAppsScript.Addon.EventObject`
  - Pros: Simple.
- `GoogleAppsScript.Events.EventObject`
  - Pros: same namespace name as `google-apps-script-events.d.ts`
  - Cons: `Events.EventObject` is lengthy a bit...?

@PopGoesTheWza Any idea?

Reply:
@mtgto Agreed - I am good with both options. If there will be no second opinion, I'd go with `Addon` namespace as it has potential to be used for other Add-ons related typings should there be any.